app/celery/tasks.py

```diff
@@ -296,7 +299,7 @@ def save_email(self,
         send_notification_to_queue(
             saved_notification,
             service.research_mode,
-            queue=template.queue_to_use()
+            queue=notification.get('queue', template.queue_to_use())
```

Comment:
```suggestion
            queue=notification.get('queue') or template.queue_to_use()
```

Reply:
I wrote it the suggested way first but I thought that using `get` would be more idiomatic, as the second parameter is the default if the `queue` key in the `notification` dictionary is `None`. What do you think? Would there be differences in code between the two?

Reply:
Answered by Antoine:
> ```{'a': None}.get('a', 42)```
> donne None en Python, t’es obligé d’utiliser un or . Ça retourne la valeur de la clé dès qu’elle existe 

That makes total sense.